README.md

```diff
@@ -33,11 +33,12 @@ In essence, LinguFlow's design and implementation method offer a structured and
 
 ### Localhost (docker)
 
-You can run LinguFlow on your local machine using [docker](https://docs.docker.com/get-docker/) compose. This setup is perfect for developing, testing LinguFlow applications, and diagnosing integration issues.
+You can run LinguFlow on your local machine using [docker compose](https://docs.docker.com/get-docker/). This setup is perfect for developing, testing LinguFlow applications, and diagnosing integration issues.
```

Comment:
https://docs.docker.com/compose/install/     

The above link seems better.

Reply:
Thank you. I will change the link.

Reply:
PTAL Again~